lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -32,7 +32,14 @@
   name:
     description:
       - Name of the security group.
-    required: true
+      - One of and only one of I(name) or I(group_id) is required.
```

Comment:
Looks good, but right now group_id can't be used with state present.
`"msg": "state is present but the following are missing: name"`